ppml/trusted-machine-learning/Dockerfile

```diff
@@ -0,0 +1,161 @@
+ARG BASE_IMAGE_NAME=intelanalytics/bigdl-ppml-gramine-base
+ARG BASE_IMAGE_TAG=2.3.0-SNAPSHOT
+ARG SPARK_VERSION=3.1.3
+ARG TINI_VERSION=v0.18.0
+ARG JDK_VERSION=8u192
+ARG JDK_URL
+ARG SPARK_JAR_REPO_URL
+ARG SCALA_VERSION=2.12
+
+
+# Stage.1 Spark & Hadoop
```

Comment:
copy from trusted big data image directly?

Reply:
It's based on trusted-bigdata Dockerfile. Some unnecessary content has been removed.

Reply:
You can directly copy complied dependency from bigdata release like:

```
ARG BIGDATA_IMAGE_NAME=bigdl-ppml-trusted-bigdata-gramine
ARG BIGDATA_IMAGE_TAG=2.3.0-SNAPSHOT

FROM ${BIGDATA_IMAGE_NAME}:{BIGDATA_IMAGE_TAG}
......
```

@SEULiyao